src/components/views/donate/OnTime/OneTimeDonationCard.tsx

```diff
@@ -279,6 +300,23 @@ const CryptoDonation: FC = () => {
 		}
 	};
 
+	useEffect(() => {
+		if (
+			amount > selectedTokenBalance &&
+			selectedOneTimeToken?.chainType === ChainType.EVM
+		) {
+			setAmountError(true);
+		} else {
+			setAmountError(false);
+		}
+	}, [selectedTokenBalance, amount, selectedOneTimeToken?.chainType]);
+
+	const amountErrorText = useMemo(() => {
+		const totalAmount = formatCrypto(amount + gasfee, tokenDecimals);
+		const tokenSymbol = selectedOneTimeToken?.symbol;
+		return `You need to keep at least ${totalAmount} ${tokenSymbol} in our wallet to pay the network fees for this transaction. Please enter a lower amount.`;
```

Comment:
Here you need to provide language string, it would be hard but you will manage it ;)